docs/src/docs/asciidoc/user/tiering.adoc

```diff
@@ -88,31 +179,98 @@ With the above into account, the following possibilities are valid configuration
 
 * heap + offheap
 * heap + offheap + disk
-* heap + disk
 * heap + offheap + clustered
+* heap + disk
 * heap + clustered
 
-== Persistent tiers
+Here is an example using heap, offheap and clustered.
 
-The _disk_ and _clustered_ tiers introduce the concept of persistence.
-It means that the data they contain can survive a restart of the Java application hosting the cache instance.
+[source,java,indent=0]
+----
+include::{sourcedir33}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/Tiering.java[tag=threeTiersCacheManager]
```

Comment:
See the other comment on an include with callouts that is not followed by their description.